pkgs/pythonPackages/libpartition.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs
+, lib
+, buildPythonPackage
+, contrailPythonBuild
+, contrailVersion
+, gevent
+, kazoo
+, pysandesh
+, pycassa
+, consistent_hash
+}:
+
+with pkgs.lib;
+
+buildPythonPackage {
```

Comment:
Does `buildPythonPackage` add a `meta` attribute set automatically? If not could you perhaps add one? I'm mainly asking because it would be nice to have a description for all of our packages.